Rakefile

```diff
@@ -37,6 +37,24 @@ RSpec::Core::RakeTask.new("spec:integration") do |t|
   t.pattern = "spec/integration/**/*_spec.rb"
 end
 
+desc "Run minimal RSpec code examples in spec/integration"
+  RSpec::Core::RakeTask.new("spec:integration:acceptance") do |t|
```

Comment:
The indentation is wrong here, it should be aligned with the beginning of the line.